src/app/shared/components/notification-card/notification-card.component.ts

```diff
@@ -3,8 +3,8 @@ import { Icon, IconSize } from '../../enums/icon'
 import { Notification } from '../../models/notification'
 import { StatusCode } from '../../enums/status-code'
 
-const INFO_COLOR: string = 'var(--blue-color)'
-const WARNING_COLOR: string = 'var(--yellow-color)'
+const INFO_COLOR: string = 'var(--info-color)'
+const WARNING_COLOR: string = 'var(--attention-color)'
```

Comment:
Here the constant name is `WARNING_COLOR` but var name is `--attention-color`, maybe we can rename the constant to be `ATTENTION_COLOR`

Reply:
In my opinion the constant refers to the actual fact that we want to indicate the notification `WARNING` level color. So to me it would seem the constant reflects the color that relates to it.

Reply:
The `WARNING` part of the constant refers to the status code of the notification. The reason that `INFO` does not match a status code is that it is an aggregation of `GOOD` and `UNKNOWN`. I would like to see these constants gone, such that the styling is handled in the SCSS file. I didn't do it in this PR to limit the number of changes.